Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -164,17 +164,25 @@ a common practice for those looking to target certain workloads onto faster
 
 The general rules for deciding how many PGs your pool(s) should contain is:
 
-* Less than 5 OSDs set pg_num to 128
-* Between 5 and 10 OSDs set pg_num to 512
-* Between 10 and 50 OSDs set pg_num to 1024
+* Fewer than 5 OSDs set `pg_num` to 128
+* Between 5 and 10 OSDs set `pg_num` to 512
+* Between 10 and 50 OSDs set `pg_num` to 1024
 
 If you have more than 50 OSDs, you need to understand the tradeoffs and how to
 calculate the pg_num value by yourself. For calculating pg_num yourself please
-make use of [the pgcalc tool](http://ceph.com/pgcalc/).
-
-If you're already using a pool it is generally safe to [increase its PG
-count](#setting-pg-count) on-the-fly. Decreasing the PG count is not
-recommended on a pool that is in use. The safest way to decrease the PG count
+make use of [the pgcalc tool](https://old.ceph.com/pgcalc/).
+
+Note that two Ceph options are in force for each Ceph pool, `pg_num` and
+`pgp_num`.  The number of PGs to _create_ for a given pool is indicated by
+`pg_num`, and the number of PGs on which to _place_ data is indicated
+by `pgp_num`.  In steady-state operation these values should be the same for a
+given pool.
+
+It is generally safe to [increase the PG count](#setting-pg-count) for an
+existing pool on-the-fly.  Ceph releases beginning with Nautilus will
+automatically increment `pgp_num` to match `pg_num`, and support reducing
+the number of PGs in a pool in place.  With Ceph releases older than Nautilus 
```

Comment:
Since Nautilus isn't supported anymore, we can remove this section on how to deal with older than Nautilus.

Reply:
Sure, I didn't know what people might still be using with Rook.